Jenkinsfile

```diff
@@ -47,12 +48,21 @@ node {
       defraUtils.waitForQualityGateResult(timeoutInMinutes)
     }
     stage('Push container image') {
-      defraUtils.buildAndPushContainerImage(regCredsId, registry, imageName, containerTag)
+      defraUtils.buildAndPushContainerImage(regCredsId, registry, repoName, containerTag)
     }
     if (pr == '') {
+       stage('Verify version incremented') {
```

Comment:
This needs to go in the `else` block.  We only want this to run if it's a PR build so `pr == ''` means not a PR.  It seems our pipelines aren't consistent when we've written this as some are checking for `pr == ''` while others have `pr != ''` and have reversed the logic.